parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -128,7 +129,9 @@ AGENT_ARTIFACTS_CONFIG_PLACEHOLDER
   owner: "root"
   content: |
     #!/bin/bash
-
+    source /opt/azure/containers/provision_source.sh
+    apt_get_update
+    retrycmd_if_failure 20 10 120 apt-get install -y iptables iproute2 socat util-linux mount ebtables ethtool init-system-helpers
```

Comment:
Are we confident that all of these packages will reliably install in under 2 mins? (speaking to the 120 second timeout)

Reply:
Yes most of them should already be there so it's very quick. In my tests it took under 10 seconds. I can increase the timeout if we want to be sure.